docs/configuration/sendrecvoptions.rst

```diff
@@ -155,5 +178,29 @@ If they are installed on the receiving side as well as the sending side, propert
 Below is an incomplete list of problematic properties.
 If you have another entry for this list, please open an issue or pull request for this page.
 
+Mount behaviour
+---------------
+
 * ``mountpoint``
 * ``canmount``
+* ``overlay``
+
+Encryption
+----------
+
+If the source datasets are encrypted but you do **not** set ``encrypted`` or ``raw`` in the :ref:`send options<job-send-options>` to ``true`` on the sending side (i.e. you do an unencrypted send) **and** you enable property replication, you will have to inherit the following properties on the :ref:`receiving side<job-recv-options>`:
+
+* ``keylocation``
+* ``keyformat``
+* ``encryption``
+
+Systemd
+-------
+
+If you use systemd, you may want to look at the properties in the `zfs-mount-generator man page <https://manpages.debian.org/buster-backports/zfsutils-linux/zfs-mount-generator.8.en.html>`_ for the ZFS releases on your machines.
```

Comment:
I'd prefer the openzfs man pages ( https://openzfs.github.io/openzfs-docs/man/8/zfs.8.html ) but I suppose the tool doesn't have a page there?

Reply:
Yup, I tried.